Makefile

```diff
@@ -120,17 +120,13 @@ perlcritic: tools/lib/
 test-unused-modules:
 	tools/detect_unused_modules
 
-.PHONY: test-deleted-renamed-referenced-modules
-test-deleted-renamed-referenced-modules:
-	tools/test_deleted_renamed_referenced_modules `git diff --name-only --exit-code --diff-filter=DR $$(git merge-base master HEAD) | grep '^tests/*'`
-
-.PHONY: test-deleted-renamed-testdata
-test-deleted-renamed-testdata:
-	tools/test_deleted_renamed_testdata `git diff --name-only --exit-code --diff-filter=DR $$(git merge-base master HEAD) | grep '^test_data/*'`
+.PHONY: test-deleted-renamed-referenced-files
+test-deleted-renamed-referenced-files:
+	tools/test_deleted_renamed_referenced_files `git diff --name-only --exit-code --diff-filter=DR $$(git merge-base master HEAD) | grep '^test*'`
 
 .PHONY: detect-nonexistent-testdata
 detect-nonexistent-testdata:
-	tools/detect_nonexistent_testdata `git diff --exit-code $$(git merge-base master HEAD) | grep '+  $include: test_data/' | grep 'yaml$$' | awk '{print $$3}'`
+	export PERL5LIB=${PERL5LIB_} ; tools/detect_nonexistent_testdata `git diff --name-only --exit-code $$(git merge-base master HEAD) | grep '^schedule/*'`
```

Comment:
As we have fixed similar issue recently, let's filter out deleted schedules. We also could have checked yaml/yml files only in schedule directory, even though there should be nothing eles

Reply:
Good point for the deleted schedules. The second sentence confuses me a bit. You mean, I should add "| grep "ml$" at the end of line 129 ?

Reply:
We could have `'^schedule/.*\.y.?ml$'` in single grep instead of 2 of them.

Reply:
done